akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/BehaviorSetup.scala

```diff
@@ -66,11 +72,16 @@ private[akka] final class BehaviorSetup[C, E, S](
   def selfClassic: ClassicActorRef = context.self.toClassic
 
   private var mdcPhase = PersistenceMdc.Initializing
+  
+  private val internalLogger = 
```

Comment:
there is also some more logging in `actualSignalHandler` in EventSourcedBehaviorImpl that should also use the internal